src/types.ts

```diff
@@ -148,6 +171,20 @@ export interface ElectronDownloadRequestOptions {
    * @defaultValue the OS default temporary directory via [`os.tmpdir()`](https://nodejs.org/api/os.html#ostmpdir)
    */
   tempDirectory?: string;
+  /**
+   * Controls the cache read and write behavior.
+   *
+   * When set to either `ReadOnly` or `Bypass` the caller is responsible
+   * for cleaning up the returned file path once they are done using
+   * it. E.g. `fs.remove(path.dirname(pathFromElectronGet))`
+   *
+   * When set to either `WriteOnly` or `ReadWrite` (the default) the caller
+   * should not move or delete the file path that is returned as the path
+   * points directly to the disk cache.
+   *
+   * Defaults to `ElectronDownloadCacheMode.ReadWrite`.
+   */
+  cacheMode?: ElectronDownloadCacheMode;
```

Comment:
```suggestion
  /**
   * Controls the cache read and write behavior.
   *
   * When set to either {@link ElectronDownloadCacheMode.ReadOnly | ReadOnly} or
   * {@link ElectronDownloadCacheMode.Bypass | Bypass}, the caller is responsible
   * for cleaning up the returned file path once they are done using it
   * (e.g. via `fs.remove(path.dirname(pathFromElectronGet))`).
   *
   * When set to either {@link ElectronDownloadCacheMode.WriteOnly | WriteOnly} or
   * {@link ElectronDownloadCacheMode.ReadWrite | ReadWrite} (the default), the caller
   * should not move or delete the file path that is returned as the path
   * points directly to the disk cache.
   *
   * This option cannot be used in conjunction with {@link ElectronDownloadRequestOptions.force}.
   *
   * @defaultValue {@link ElectronDownloadCacheMode.ReadWrite}
   */
```

nit: use fancy `@link` tags for better intellisense

Reply:
had a typo here, @_@

[641415f](https://github.com/electron/get/pull/267/commits/641415fba70d98aa28d025d5c9b201544c7aac2e)